akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -616,7 +616,7 @@ Note that the stashed commands are kept in an in-memory buffer, so in case of a
 processed.
 
 * Stashed commands are discarded if the actor (entity) is passivated or rebalanced by Cluster Sharding.
-* Stashed commands are discarded if the actor is restarted (or stopped) due to that an exception was thrown from processing a command or side effect after persisting.
+* Stashed commands are discarded if the actor is restarted (or stopped) in case an exception was thrown from processing a command or side effect after persisting.
```

Comment:
While the original wording didn't make sense, this loses the causality... perhaps we can do something like:
```suggestion
* Stashed commands are discarded if the actor is restarted (or stopped) because an exception was thrown while processing a command or side effect after persisting.
```
Also perhaps 'when' would be better than 'if' here?

Reply:
I gave it a go and rephrased it. Please have a special look at the preserving part, I am not sure if the causality I emphasised now is actually correct.